package.json

```diff
@@ -58,18 +60,19 @@
     "doctoc": "^1.3.0",
     "esdoc": "^0.5.2",
     "http-server": "^0.9.0",
+    "lodash": "^4.17.4",
     "mocha": "^3.3.0",
     "sinon": "~1.14.1",
     "standard-version": "^4.0.0",
     "ts-node": "^3.0.2",
+    "tslib": "^1.7.1",
     "tslint": "^5.1.0",
     "tslint-language-service": "^0.9.2",
     "typescript": "~2.3.4"
   },
   "peerDependencies": {
-    "lodash": "4.x"
+    "lodash": "4.x",
+    "tslib": "^1.7.1"
```

Comment:
tslib should be a dependency. The user shouldn't be required to install a library dependent on how this library is built. Lodash should be a peer and tslib should be a dependency.